Flutter/datagrid/filtering.md

```diff
@@ -1325,4 +1325,68 @@ Widget build(BuildContext context) {
 {% endhighlight %}
 {% endtabs %}
 
-<img alt="flutter datagrid shows filter icon when hover the header cell" src="images/filtering/flutter-datagrid-show-filter-icon-on-hover.gif"/>
\ No newline at end of file
+<img alt="flutter datagrid shows filter icon when hover the header cell" src="images/filtering/flutter-datagrid-show-filter-icon-on-hover.gif"/>
+
+## Show the column header icons when hovering the header cell
+
+To show a column header icons,such as sort and filter icons when the mouse hovers over a column header in a DataGrid, set the `SfDataGrid.showColumnHeaderIconOnHover` property to true. Note that this feature is only available on web and desktop platforms.

```

Comment:
To display column header icons, such as sort and filter icons, when the mouse hovers over a column header, set the `SfDataGrid.showColumnHeaderIconOnHover` property to true. This feature is available only on web and desktop platforms.

Reply:
@ashok-kuvaraja - added the content